pkg/api/types.go

```diff
@@ -2785,6 +2802,12 @@ type DeleteOptions struct {
 	// finalizer will be added to/removed from the object's finalizers list.
 	// +optional
 	OrphanDependents *bool
+
+	// Whether and how garbage collection will be performed.
+	// Defaults to DeletePropagationDefault.
+	// Either this field or OrphanDependents may be set, but not both.
+	// +optional
+	PropagationPolicy *DeletePropagationPolicy
```

Comment:
go-restful does not provide enough information for the generator to figure out this type alias. I suggest as a workaround, you either:

1. Add your new type to the map in this function: https://github.com/kubernetes/apiserver/blob/master/pkg/endpoints/installer.go#L1045
2. Do not use alias for types that can be used in query parameters.

I would find a better way to fix this, but that involve some changes in go-restful and it could take a while.